source/options.tsx

```diff
@@ -12,7 +12,8 @@ import {perDomainOptions} from './options-storage';
 import * as domFormatters from './github-helpers/dom-formatters';
 
 function parseDescription(description: string): DocumentFragment {
-	const descriptionElement = <span>{description}</span>;
+	const descriptionElement = <span/>;
+	descriptionElement.innerHTML = description.replaceAll('\\`', '`').replaceAll('<iframe>', '&lt;iframe&gt;');
```

Comment:
If this is about ``<code>\`</code>``, can you also try replacing that with:

	`` ` ``

Which is rendered as `` ` ``. I don't know if this `.replaceAll` is better than having to support that string in `parseBackticks()`

---

Also there's a `<details>` too I think